iroh-net-report/src/reportgen/hairpin.rs

```diff
@@ -163,7 +163,7 @@ impl Actor {
 
         socket
             .send_to(
-                b"tailscale netcheck; see https://github.com/tailscale/tailscale/issues/188",
+                b"tailscale net_report; see https://github.com/tailscale/tailscale/issues/188",
```

Comment:
I am not sure we can change this, needs checking

Reply:
sorry 😅 I was too eager

Reply:
we can change this and allows us to make this more generic